cmd/kubetoken/kubetoken.go

```diff
@@ -54,25 +54,36 @@ You can find more information on 'ExecCredential' and 'client side authenticatio
 	cmd.RunE = func(_ *cobra.Command, args []string) error {
 		ctx := context.Background()
 
-		err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{
+		k8sClientProvider := okteto.NewK8sClientProvider()
+		okClientProvider := okteto.NewOktetoClientProvider()
```

Comment:
I think this way is more readable to init a struct with several params. If we forgot to set one of them, the default ones will be used so we could remove `withK8sClientProvider(k8sClientProvider)` and the code still works. I think it's a good way of reducing panics. WDYT?